tests/Rules/Fixture/laravel-view-function.php

```diff
@@ -47,3 +47,14 @@
 ];
 
 view('foo', $fooBar);
+
+class MyDto implements \Illuminate\Contracts\Support\Arrayable {
+    /**
+     * @return array{foo: string}
+     */
+    public function toArray() {
+        return ['foo' => 'bar'];
+    }
+}
+
+view('foo', new MyDto());
```

Comment:
I think there is an unhanded case on `View::make($name, $params)`

Reply:
it does not, just found out. Replaced with view() and there more errors to fix now 😄 

> though it should bot be part of this PR.

Yes, no need. Just adding it somewhere so I do not forget about it

Reply:
One more weird case with the 2d array, it seems that if array of array has a string or float is does 
`Offset 'tvaNumber' does not exist on array|string.`

```php
        $invoice = [
            'foo'      => $foo->toArray(),
            'bar'     => $bar->toArray(),
            //'total'     => $this->total,
            //'reference' => $this->reference,
        ];

            $data = [
                'invoice' => $invoice,
                'pageTitle' => 'Invoice',
            ];
```
template: `{{$invoice['foo']['tvaNumber']}}`

If I un-comment one one of the two lines it says the error. What is weird is that is says `array|string` and this type is very wrong

Reply:
This does not sound like something related to Bladestan. use `\PHPStan\dumpType($data);`  to probe what type it gets resolved to`.